### DISCUSSION TITLE
Unable to use Copilot inside VDI/Citrix - Login github

### DISCUSSION THREAD

**Discussion by AndreuSCK** [community/community#32073] [Copilot Conversations]
Hi, There is any way I could use copilot? Vscode does not let me login:
-The Github website opens, I login, it appears a screen "you may close the screen now and return to vscode", then in vscode it appears:
Allow an extension to open this URI? vscode://vscode.github.authenti(......) open/cancel
I click open, and after that it keeps thinking forever.

My guess is that my vscode does not have perms to execute web protocols? There is any other way I could use copilot/login?  Thanks

**Comment by 3kh0** [comment]
This might be because you do not have access to the GitHub copilot program, if not, check your application permissions and your default browser.


*If this helped please mark it as the answer*

**Reply by AndreuSCK** [reply]
Hi, I have access to Copilot, I use it on pc, but im not able to run it inside the dvi,
 there is any way to check what permissions has vscode? Default browser affects that?

**Reply by 3kh0** [reply]
Default browser does affect that, could you also attach your VSCode logs?
